server/lobbyconnection.py

```diff
@@ -586,11 +590,18 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
         old_player = self.player_service.get_player(self.player.id)
         if old_player:
             self._logger.debug("player {} already signed in: {}".format(self.player.id, old_player))
-            if old_player.lobby_connection:
-                await old_player.lobby_connection.send_warning("You have been signed out because you signed in elsewhere.", fatal=True)
-                old_player.lobby_connection.game_connection = None
-                old_player.lobby_connection.player = None
-                self._logger.debug("Removing previous game_connection and player reference of player {} in hope on_connection_lost() wouldn't drop her out of the game".format(self.player.id))
+            if old_player.lobby_connection is not None:
```

Comment:
Can be `old_player.lobby_connection` as `0` or `False`? If not, i'd use `if old_player.lobby_connection`, no need to check for None explicit, it's easier to read

Reply:
I'm simply following the example in the `weakref` docs. They check for `None` explicitly, so that's what I'm doing too.

https://docs.python.org/3/library/weakref.html#weak-reference-objects